content/en/examples/controllers/job-backoff-limit-per-index-failindex.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-backoff-limit-per-index-failindex
+spec:
+  completions: 4
+  parallelism: 2
+  completionMode: Indexed
+  backoffLimitPerIndex: 1
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: main
+        image: python
+        command:
+          # The script:
+          # - fails the Pod with index 0 with exit code 1 which result in retry,
+          # - fails the Pod with index 1 with exit code 42 which results
+          #   in failing the index without retry.
+          # - succeeds Pods with any other index.
+          - python3
+          - -c
+          - |
+            import os, sys
+            index = int(os.environ.get("JOB_COMPLETION_INDEX"))
+            if index == 0:
+              sys.exit(1)
+            elif index == 1:
+              sys.exit(42)
+            else:
+              sys.exit(0)
```

Comment:
nit: consider adding a `time.sleep()` before successful exit.

Reply:
Yeah, I was thinking about it, I see we are not consistent about it. However, I'm not clear what it will improve in this context. For example sleep(1s) would not be noticed probably as we anyway recommend to examine the pods after around 15s.

Reply:
Ok, let me keep it as is then.